plugins/parsers/bindata/README.md

```diff
@@ -0,0 +1,50 @@
+# BinData
+
+The "BinData" parser translates binary records consisting of multiple fields into Telegraf metrics. It supports:
+
+* Little- and Big-Endian encoding
+* bool, int8/uint8, int16/uint16, int32/uint32, int64/uint64, float32/float64 field types
+* UTF-8 and ASCII-encoded strings
+* unix, unix_ms, unix_us and unix_ns timestamp
+
+### Configuration
+
+```toml
+[[inputs.mqtt_consumer]]
+  name_override = "drone_status"
+
+  ## Data format to consume.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
+  data_format = "bindata"
+
+  ## Numeric fields endiannes, "be" or "le"
+  bindata_endiannes = "be"
```

Comment:
How about adding `machine` and make it the default? That is, by default use the endianess of the machine that we have to detect.

Reply:
Sure, can be done that way.

Reply:
But, what if the measurement arrives from a  different machine which has different than the endianness of the machine where Telegraf runs?

Reply:
Then you have to specify the endianness (which can be done). However, it is not uncommon to have machines with the same endianness, so a sensible default should be chosen IMO.

Reply:
"be" is the default, and I commented the line out in README.md but can't see the update here.